cloudbuild/nightly/scripts/python-scripts/assert_table_count.py

```diff
@@ -52,14 +69,19 @@ def assert_total_row_count(client, project_name, dataset_name, source_table_name
             raise AssertionError("Destination Row count is less than Source Row Count")
 
 
-def assert_unique_key_count(client, project_name, dataset_name, source_table_name,
+def assert_unique_key_count(bq_client, storage_client, project_name, dataset_name, source,
                             destination_table_name,
+                            mode,
                             is_exactly_once):
-    source_unique_key_count = get_unique_key_count(client, project_name, dataset_name,
-                                                   source_table_name)
+    source_unique_key_count = 0
+    if mode == "unbounded":
+            source_unique_key_count = get_total_row_count_unbounded(storage_client, source)
+    else:
```

Comment:
Do we not check unique key for unbounded test?

Reply:
All rows in source csv file are unique hence to identify unique rows count for unbounded source, we use the get_total_row_count function.

Irrespective of the source type, we validate if the sink works as intended and the no. of unique rows are same in the destination table.